README.md

```diff
@@ -1,73 +1,83 @@
 # Tarantool
 
-[![Build Status][travis-badge]][travis-url]
-[![Build Status][gitlab-ci-badge]][gitlab-ci-url]
+[![Actions Status][actions-badge]][actions-url]
 [![Code Coverage][coverage-badge]][coverage-url]
 [![Telegram][telegram-badge]][telegram-url]
-[![Slack][slack-badge]][slack-url]
-[![Gitter][gitter-badge]][gitter-url]
-[![Google Groups][groups-badge]][groups-url]
+[![GitHub Discussions][discussions-badge]][discussions-url]
+[![Stack Overflow][stackoverflow-badge]][stackoverflow-url]
 
-https://tarantool.io/en/
-
-Patch submissions and discussion of particular patches https://lists.tarantool.org/mailman/listinfo/tarantool-patches/
-
-General development discussions https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/
+https://www.tarantool.io/en/
 
 Tarantool is an in-memory database and application server.
```

Comment:
IMHO, it would be great to highlight somewhere (preferably at the beginning of the README) the license under which Tarantool is distributed.

Reply:
Good idea. Added.